latch_cli/services/init/init.py

```diff
@@ -261,26 +297,16 @@ def init(
             return False
 
     base_image = latch_constants.base_image
-    if cuda and opencl:
-        raise ValueError("Latch does not support both CUDA and OpenCL yet")
-    elif cuda:
-        base_image = base_image.replace("latch-base", "latch-base-cuda")
-    elif opencl:
-        base_image = base_image.replace("latch-base", "latch-base-opencl")
-
-    config = LatchWorkflowConfig(
-        latch_version=get_distribution("latch").version,
-        base_image=base_image,
-        date=datetime.now().isoformat(),
-    )
+    if base_image_type is not None:
+        base_image = base_image.replace("latch-base", f"latch-base-{base_image_type}")
 
-    (pkg_root / ".latch").mkdir(exist_ok=True)
-
-    with open(pkg_root / latch_constants.pkg_config, "w") as f:
-        f.write(json.dumps(asdict(config)))
+    if template == "docker":
```

Comment:
you shouldn't be able to specify a `base_image` with a template that isn't empty, so have a check to see if both are provided and throw an error if the template is nontrivial or something